README.md

```diff
@@ -38,9 +38,51 @@ For more information about the default configuration, see [Default Secure Landin
+
+    - If you don't have an IBM Cloud account, [create one](https://cloud.ibm.com/docs/account?topic=account-account-getting-started).
+    - If you have a Trial or Lite account, [upgrade your account](https://cloud.ibm.com/docs/account?topic=account-upgrading-account).
+
+### Configure your IBM Cloud account for Secure Landing Zone
+
+1.  Log in to [IBM Cloud](https://cloud.ibm.com) with the IBMid you used to set up the account. This IBMid user is the account owner and has full IAM access.
+1.  [Complete the company profile](https://cloud.ibm.com/docs/account?topic=account-contact-info) and contact information for the account. This profile is required to stay in compliance with IBM Cloud Financial Service profile.
+1.  [Enable the Financial Services Validated option](https://cloud.ibm.com/docs/account?topic=account-enabling-fs-validated) for your account.
+1.  Enable virtual routing and forwarding (VRF) and service endpoints by creating a support case. Follow the instructions in  [enabling VRF and service endpoints](https://cloud.ibm.com/docs/account?topic=account-vrf-service-endpoint&interface=ui#vrf).
+
+### Set up account access (Cloud IAM)
+
+1.  If you are provisioning resources manually, create an IBM Cloud [API key](https://cloud.ibm.com/docs/account?topic=account-userapikey#create_user_key). The user who owns this key must have the Administrator role.
+1.  Require users in your account to use [multifactor authentication (MFA)](https://cloud.ibm.com/docs/account?topic=account-account-getting-started#account-gs-mfa).
+1.  [Set up access groups](https://cloud.ibm.com/docs/account?topic=account-account-getting-started#account-gs-accessgroups).
+    User access to IBM Cloud resources is controlled by using the access policies that are assigned to access groups. For IBM Cloud Financial Services validation, all IAM users must not be assigned direct access to any IBM Cloud resources.
+
+    Select **All Identity and Access enabled services** when you assign access to the group.
+
+### (Optional) Set up IBM Cloud Hyper Protect Crypto Services
+
+For Key Management services, you can use IBM Cloud Hyper Protect Crypto Services. Create an instance before you create the Secure Landing Zone.
+
+1.  Create the service instance:
+
+    1.  (Optional) [Create a resource group](https://cloud.ibm.com/docs/account?topic=account-rgs&interface=ui) for your instance.
+    1.  On the Hyper Protect Crypto Services (https://cloud.ibm.com/catalog/services/hyper-protect-crypto-services) details page, select a plan.
+    1.  Complete the required details that are required and click **Create**.
+
+1.  Initialize Hyper Protect Crypto Services:
+
+    To initialize the provisioned Hyper Protect Crypto Service instance, follow the steps in [Getting started with IBM Cloud Hyper Protect Crypto Services](https://cloud.ibm.com/docs/hs-crypto?topic=hs-crypto-get-started) and perform the quick initialization.
```

Comment:
@vburckhardt I don't see any mention of a quick initialization. I see 3 options in the docs about initialization. We should point to one of those, or use the term that they can search for on the docs page.

Reply:
Yea looks like they changed to the docs.  I would just take the "and perform the quick initialization" out